### ISSUE TITLE
[Bug]: Page Header commands do not retain order set in settings

### ISSUE THREAD

**Issue by Triquetra** [phibr0/obsidian-commander#7]
### Describe the bug

given order of commands x, y, z
the app periodically changes the order of y and z to the app's preferred order x, z, y

in my use case, y = "Navigate forward" and z = "Navigate back"

### Steps to reproduce

set your own preferred order of page header commands, watch the app blend them at it it's leasure

### Expected Behavior

commands stay in the order which the user sets in the settings

### Additional context

I'm using these particular commands in conjunction with the Pane Relief plugin for per-pan navigation. This works great with Commander, except for the random command reordering. The Customizable Page Header and Title Bar plugin handled these two commands without issue and without random reordering.

### Operating system

Windows

**Comment by phibr0** [comment]
Does this still happen after the recent Changes?

**Comment by Triquetra** [comment]
Yes, it is still happening. But it is no longer an issue for me at the moment because I am using the now default per tab navigation built into Obsidian 1.0. So I will now only be using this plugin to add one command to the page header, and the order of one command does not matter... :)
